packages/ripple/src/getRippleGlobalNamespace.ts

```diff
@@ -0,0 +1,43 @@
+import { Options } from './utils';
+
+interface ModuleType {
+  '@leafygreen-ui/ripple': {
+    registeredRippleElements: Map<HTMLElement, Options>;
+    setRippleListener: boolean;
+  };
+}
+
+type LGWindow = Window &
+  typeof globalThis & {
+    __LEAFYGREEN_UTILS__: {
+      modules: ModuleType;
+    };
+  };
+
+export function getRippleGlobalNamespace() {
+  // Handles SSR
+  if (typeof window === 'undefined') {
+    return;
+  }
+
+  const defaultRippleParams = {
+    setRippleListener: false,
+    registeredRippleElements: new Map(),
+  };
+
+  if ((window as LGWindow).__LEAFYGREEN_UTILS__) {
+    (window as LGWindow).__LEAFYGREEN_UTILS__.modules[
+      '@leafygreen-ui/ripple'
+    ] = defaultRippleParams;
+  } else {
+    const rippleModule = {
+      '@leafygreen-ui/ripple': defaultRippleParams,
+    };
+
+    (window as LGWindow).__LEAFYGREEN_UTILS__ = { modules: rippleModule };
+  }
```

Comment:
Fully optional, just thinking aloud, using https://github.com/tc39/proposal-logical-assignment let's us have some fun:

```suggestion
  (window as LGWindow).__LEAFYGREEN_UTILS__ ??= { modules: {} };
  (window as LGWindow).__LEAFYGREEN_UTILS__.modules[
      '@leafygreen-ui/ripple'
    ] = defaultRippleParams;
```

Reply:
So much cleaner